File: src/app/gkill/api/req_res/get_texts_by_target_id_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/app/gkill/api/message"
	"github.com/mt3hr/gkill/src/app/gkill/dao/reps"
)

type GetTextsByTargetIDResponse struct {
	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	Texts []*reps.Text `json:"texts"`
}

File: src/classes/api/req_res/get-texts-by-target-id-response.ts
'use strict'

import { Text } from '@/classes/datas/text'
import { GkillAPIResponse } from '../gkill-api-response'

export class GetTextsByTargetIDResponse extends GkillAPIResponse {

    texts: Array<Text>

    constructor() {
        super()
        this.texts = new Array<Text>()
    }

}


